packages/docs/docs/concepts/progressive-loading/requirements.md

```diff
@@ -0,0 +1,41 @@
+---
+id: requirements
+---
+
+# Requirements and Configuration for Progressive Loading
+
+The progressive loading will improve stack image display just given support
+of HTJ2K progressive resolution encoded data, while volumetric data is improved
+in time to first volume for all back ends not using customized streaming loaders.
+However, the support of different types of reduced resolution and streaming
+responses is quite varied between DICOMweb implementations. Thus, this
+guide provides some additional details on how to configure various options,
+as well as how to modify the default load order for different requirements.
+
+## Separate Path or Argument Partial Resolution
+
+One way of configuration for partial image resolution loading is to add
+a separate path. This is done with the framesPath argument, which replaces
+the /frames/ part of the path with an alternate path.
+
+Alternatively, or in addition, the urlArguments can be added.
+
+```
+retrieveConfiguration: {
+    'default-lossy': {
```

Comment:
I think a separate page for retireveConfiguration is needed in this progressive-loading folder. To discuss different Enums (e.g., default-lossy etc.)

Reply:
is this an enum or it can be named anything?

Reply:
I feel like at the start of this new request section, we should talk about that there are two ways to load there is a streaming and there is a byte range (I guess?)